src/Model/Contact.php

```diff
@@ -1151,6 +1151,11 @@ public static function createFromProbe($uid, $url, $interactive = false, $networ
 
 		Addon::callHooks('follow', $arr);
 
+		if (empty($arr)) {
+			$result['message'] = L10n::t('Contact cannot be added.');
```

Comment:
Can you please be more specific about the error? Ask to check the credentials if it's an addon-provided network for example.

Reply:
Then the message had to be generated by the addon.

Reply:
It is a solution, but this check is after the Addon hook, so you can just write something general about addon-provided networks and be done with it.

Reply:
I don't have a clue for a nice message.

Reply:
The contact could not be added. It is possible you are attempting to add a contact that requires an Addon that is not enabled on this system.

Reply:
Done